CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## 77.0.0-SNAPSHOT - unreleased
 
+### 🎁 New Features
+
+* `DashCanvasView` and `DashContainerView` components now have a public `@bindable` `titleDetails`
+  property on their models to support displaying additional information in the title bar of these
+  components.  `titleDetails` is not persisted, and is expected to be set programmatically by the
```

Comment:
Can't think of a better name for this, but also not super attached to the name -- if anyone has any better suggestions....